redis/server.cc

```diff
@@ -50,6 +50,23 @@ redis_server::redis_server(distributed<service::storage_proxy>& proxy, distribut
     , _memory_available(_max_request_size)
     , _auth_service(auth_service)
 {
```

Comment:
Please use a different title for each patch. If this patch is about statistics, title it something like "Redis API: add statistics".

Reply:
OK. I got it.